src/index.js

```diff
@@ -1,62 +1,80 @@
-import { getScores, addScore } from './apiHandler';
-import './styles.css';
+import { getScores, addScore } from './apiHandler'
+import './styles.css'
 
-const listBox = document.getElementById('listbox');
-const form = document.getElementById('form');
-const name = document.getElementById('name');
-const score = document.getElementById('score');
-const refresh = document.getElementById('refresh');
-const alertBox = document.getElementById('alert-box');
-const alertBox2 = document.getElementById('alert-box2');
+const listBox = document.getElementById('listbox')
+const form = document.getElementById('form')
+const name = document.getElementById('name')
+const score = document.getElementById('score')
+const refresh = document.getElementById('refresh')
+const alertBox = document.getElementById('alert-box')
+const alertBox2 = document.getElementById('alert-box2')
+const loader = document.getElementById('loader')
+const listBoxDiv = document.getElementById('list-box-div')
```

Comment:
- [ ] Your Github workflow action is failing due to missing semicolons in your code, kindly fix this issue.
You can run the code below to install the linter packages in order to get tips or even auto-fixes right in your IDE. 
```shell
# For JavaScript

npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
```

```shell
# For style

npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
```